### PR TITLE
[BACKLOG-40086] Schedule Output Perspective : Allow user to browse Scheduled contents from a VFS location folder

### PR DESCRIPTION
**Pull request by singletonc**
Issue: https://hv-eng.atlassian.net/browse/BACKLOG-40086
Depends on:
- https://github.com/pentaho/pentaho-scheduler-plugin/pull/145

To also be merged with:
- https://github.com/pentaho/pentaho-platform/pull/5562
- https://github.com/pentaho/pentaho-scheduler-plugin-ee/pull/191

--
- Incorporate new filter parameter into exisiting endpoint calls for Schedule Select Folder Dialog
- Updated applicable "/folders" to "/files" endpoint calls

### CODE REVIEW

**Comment by buildguy** [comment]
<div align='center'>

[👍 Frogbot scanned this pull request and found that it did not add vulnerable dependencies.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 7m 1s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -Djs.no.sandbox -pl widgets
```
---
:ok_hand: All tests passed!
 
Tests run: **345**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/gwt-widgets/job/PR-1028/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: widgets/src/main/java/org/pentaho/mantle/client/dialogs/folderchooser/FolderTree.java [hunk 1]
```diff
@@ -150,12 +150,13 @@ public void onResponseReceived( Request request, Response response ) {
 
   @NonNull
   private String getServiceBaseUrl() {
-    return getFullyQualifiedURL() + "plugin/scheduler-plugin/api/generic-files/folders/";
+    return getFullyQualifiedURL() + "plugin/scheduler-plugin/api/generic-files/files/";
```
Comment (dcleao) [inline widgets/src/main/java/org/pentaho/mantle/client/dialogs/folderchooser/FolderTree.java:153]: All good except the `/files` path segment in the URLs, as noted in the related PR.

**Review by dcleao — CHANGES_REQUESTED** [review]

**Comment by buildguy** [comment]
<div align='center'>

[👍 Frogbot scanned this pull request and found that it did not add vulnerable dependencies.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 7m 47s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -Djs.no.sandbox -pl widgets
```
---
:ok_hand: All tests passed!
 
Tests run: **345**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/gwt-widgets/job/PR-1028/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
<div align='center'>

[👍 Frogbot scanned this pull request and found that it did not add vulnerable dependencies.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 7m 29s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -Djs.no.sandbox -pl widgets
```
---
:ok_hand: All tests passed!
 
Tests run: **345**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/gwt-widgets/job/PR-1028/3/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by dcleao — APPROVED** [review]

**Comment by buildguy** [comment]
<div align='center'>

[👍 Frogbot scanned this pull request and found that it did not add vulnerable dependencies.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 7m 48s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -Djs.no.sandbox -pl widgets
```
---
:ok_hand: All tests passed!
 
Tests run: **345**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/gwt-widgets/job/PR-1028/4/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by rmansoor — APPROVED** [review]

**Pull request merged**